<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPage</class>
 <widget class="QWizardPage" name="WizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>531</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>168</y>
       <width>56</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginLineEdit_2">
     <property name="geometry">
      <rect>
       <x>183</x>
       <y>171</y>
       <width>142</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUpPushButton">
     <property name="geometry">
      <rect>
       <x>331</x>
       <y>168</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>교사용클라이언트 로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginPushButton">
     <property name="geometry">
      <rect>
       <x>331</x>
       <y>131</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>131</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginLineEdit">
     <property name="geometry">
      <rect>
       <x>183</x>
       <y>134</y>
       <width>142</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="idFindButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>id찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pwFindButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>pw찾기</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>환영함</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>417</width>
       <height>206</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_new_pw"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>비밀번호</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="EmailCheckPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>인증 요청</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_new_pw_check"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_new_id">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>아이디</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>비밀번호 확인</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>이메일</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>인증번호</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_email_check">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="EmailCheckNumberPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>인증</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>이름</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_new_name"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="SignUpCheckButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>중복확인</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QLabel" name="SignUpLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디와비밀번호를입력해주세요.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUpPushButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>380</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>교사용 클라이언트 회원가입</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>id찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="idFindPageEmailButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>이메일 확인</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idFindPageEmailLineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>이메일 입력</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디 입력</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호 찾기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwFindPageEmailLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwFindPageIdLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>이메일 입력</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pwFindPageEmailButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>220</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호 전송</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pwFindPageIdButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>190</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디 확인</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="StudentScore_Button_1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>260</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>학생별 점수조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qnaButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>360</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Q&amp;&amp;A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QLabel" name="userNameLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>유저명</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>교사 메인화면</string>
     </property>
    </widget>
    <widget class="QLabel" name="userNameLabel_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>당신의 이름:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quploadbutton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>33</pointsize>
      </font>
     </property>
     <property name="text">
      <string>문제 출제</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mainPageCounselButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>46</pointsize>
      </font>
     </property>
     <property name="text">
      <string>상담</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTextBrowser" name="chatTextBrowser">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>381</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="chatLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>580</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="chatBackButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>580</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>상담 나가기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1대1상담(지금은 전체 채팅)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="score_check_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>문제출제</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="qOKbutton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>560</y>
       <width>89</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>출제버튼</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qbackbutton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>560</y>
       <width>89</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기버튼</string>
     </property>
    </widget>
    <widget class="QListWidget" name="qlistwidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>231</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qlineEdit_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>510</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="qlabel_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>520</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>정답</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>29</pointsize>
      </font>
     </property>
     <property name="text">
      <string>점수확인창</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="score_back_pushButton_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>530</y>
       <width>89</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="StudentScore_Table_Widget_1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>491</width>
       <height>271</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>학생이름</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>학생이름</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>학생이름</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>학생이름</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>이 름</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종 류</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>점 수</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="student_score_Button2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>530</y>
       <width>89</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>점수확인버튼</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_9">
    <widget class="QTableWidget" name="qnatable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>431</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>학생이름</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>교사이름</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>학생의질문</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>교사의답변</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="qna_pagelabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qnaback">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>600</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qnalineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
